update.js

```diff
@@ -0,0 +1,37 @@
+const request = require('request');
+const fs = require('fs');
+const execSync = require('child_process').execSync;
+const CURRENT_VERSION = require('./lib/chromedriver').version;
+
+// fetch the latest chromedriver version
+const getLatest = (cb) => {
+  request('https://chromedriver.storage.googleapis.com/LATEST_RELEASE', (err, response, body) => {
+    if (err) {
+      process.exit(1);
+    }
+    return cb(body);
+  });
+};
+
+/* Provided a new Chromedriver version such as 77.0.3865.40:
+   - update the version inside the ./lib/chromedriver helper file e.g. exports.version = '77.0.3865.40';
+   - add a git commit and tag of npm version, e.g. Bump version to 77.0.0
+   - bumps npm package version
+*/
+const writeUpdate = (version) => {
+  const helper = fs.readFileSync('./lib/chromedriver.js', 'utf8');
+  const versionExport = 'exports.version';
+  const regex = new RegExp(`^.*${versionExport}.*$`, 'gm');
+  const updated = helper.replace(regex, `${versionExport} = '${version}';`);
+  fs.writeFileSync('./lib/chromedriver.js', updated, 'utf8');
+  execSync(`git add . && git commit -m "Bump version to ${version.slice(0, 2)}.0.0" && npm version ${version.slice(0, 2)}.0.0`);
```

Comment:
This generated two commits:
![image](https://user-images.githubusercontent.com/334958/65063251-4800c100-d954-11e9-937f-5db422e3bdca.png)

We only need one.